### PR TITLE
Code --refactored

### PR DESCRIPTION
**Pull request by itsmebipul00**
Rearranged and refactored code for consistent layout. 
PS: more work to be done on responsiveness, code snippets, redundant codes and unnecessary usage of !important

### CODE REVIEW

File: Components/Aside/Aside.css [hunk 1]
```diff
@@ -47,23 +40,33 @@ article {
     margin: 0;
   }
 
-  aside li:hover {
+  /* aside li:hover {
     cursor: pointer;
     background: hsl(var(--clr-light));
-  }
+  } */
```
Comment (zoef-yt) [inline Components/Aside/Aside.css:46]: Changes:- remove commented code

File: docs.html [hunk 2]
```diff
@@ -22,52 +22,94 @@
...
-                <li><a href="#"><span aria-hidden="true">Input</span></a></li>
-                <li><a href="#"><span aria-hidden="true">List</span></a></li>
-                <li><a href="#"><span aria-hidden="true">Toast</span></a></li>
-                <li><a href="#"><span aria-hidden="true">Modal</span></a></li>
-                <li><a href="#"><span aria-hidden="true">Slider</span></a></li>
-                
-              </ul>
-            </nav>
+                <div class="margin-l-auto d-flex">
+                  <a href="https://github.com/itsmebipul00" class="linkto-github width-5rem">
```
Comment (santhoshpemmaka) [inline docs.html:28]: I think a tag opens the GitHub profile. It's a better open new tab. If possible add target="_blank"

File: Components/Navigation/Navigation.css [hunk 3]
```diff
@@ -10,18 +10,36 @@
...
+
+.nav {
+  margin-bottom: 1rem;
+}
+
 @media (max-width: 60em) {
   .mobile-nav-toggle {
     display: block;
-    position: absolute;
+    /* position: absolute; */
```
Comment (zoef-yt) [inline Components/Navigation/Navigation.css]: Change:- remove commented code

File: Components/Submenus/Submenus.css [hunk 4]
```diff
@@ -35,6 +31,7 @@
   display: none;
   background: hsl(var(--clr-white));
   top: 100%;
+  z-index: 99999999;
```
Comment (zoef-yt) [inline Components/Submenus/Submenus.css:34]: Suggestion: Don't use such high value for z-index

File: docs.css [hunk 5]
```diff
@@ -1,29 +1,12 @@
...
 
 header {
-  background: hsl(var(--clr-light));
+  background: hsl(var(--clr-white));
 }
 
 .header-container {
-  position: relative;
-  padding: 1rem 2rem;
+  /* position: relative; */
```
Comment (zoef-yt) [inline docs.css]: Changes: remove commented code.

File: prism.css [hunk 6]
```diff
@@ -207,7 +207,7 @@ div.code-toolbar {
   height: 25vh;
   width: 80%;
   overflow: auto;
-  margin-top: 1rem;
+  margin: 1rem 0;
   background-color: #fdfdfd;
```
Comment (zoef-yt) [inline prism.css:211]: Change: use var for colour.

**Review by zoef-yt — APPROVED** [review]
The changes are minor. The rest of the code looks good. You can go ahead and merge with changes.

**Review by Ranjeetk489 — APPROVED** [review]

**Pull request merged**